### PR TITLE
Table view for itwinGrid

### PR DESCRIPTION
**Pull request by DanishMehmood-bit**
image

### CODE REVIEW

File: packages/modules/imodel-browser/src/containers/ITwinGrid/ITwinGrid.tsx [hunk 1]
```diff
@@ -90,9 +105,14 @@ export const ITwinGrid = ({
   tileOverrides,
   useIndividualState,
   postProcessCallback,
+  viewMode,
 }: ITwinGridProps) => {
   const strings = _mergeStrings(
     {
+      tableColumnName: "ITwin Number",
+      tableColumnDescription: "ITwin Name",
```
Comment (raplemie) [inline packages/modules/imodel-browser/src/containers/ITwinGrid/ITwinGrid.tsx:113]: As per our documentation site, it appears that the i of iTwin is never capitalized: 
image

```suggestion
      tableColumnName: "iTwin Number",
      tableColumnDescription: "iTwin Name",
```

**Review by raplemie — CHANGES_REQUESTED** [review]
It also appears to be missing a `rush change`, which I guess our pipeline should have caught, but somehow didnt.

File: packages/modules/imodel-browser/src/types.ts [hunk 2]
```diff
@@ -91,4 +91,4 @@ export type ITwinStatus =
   | "active"
   | "inactive"
   | "trial";
-export type IModelViewType = "tile" | "cells";
+export type ViewType = "tile" | "cells";
```
[the remark this answers is not in the document]
Comment (DanishMehmood-bit) [reply packages/modules/imodel-browser/src/types.ts:94]: Done

**Review by raplemie — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**